tools/get-chrome-refresh-token.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const request = require('request');
+const http = require('http');
+
+// This script is used to generate new chrome web store access tokens and refresh tokens from the
+// chrome web store API (see docs: https://developer.chrome.com/docs/webstore/using_webstore_api/).
+// Before running this script, be sure to updated the following variables with the appropriate
+// client ID and secret values.
+const clientId = 'TODO';
```

Comment:
Two thoughts come to mind:
1. Instead of having these hardcoded in code, would it make sense to have these as command line parameters? That seems like less work than getting the secrets, editing the script, then undoing the edits, and it makes it impossible to accidentally commit the secrets
2. I'm always a fan of a comment that shows an example command line.

Reply:
Good points, updated!